analysis/tools/ccsh/src/main/kotlin/de/maibornwolff/codecharta/tools/ccsh/Ccsh.kt

```diff
@@ -125,21 +125,26 @@ class Ccsh : Callable<Void?> {
                     val currentExitCode = executeConfiguredParser(commandLine, configuredParser)
                     if (currentExitCode != 0) {
                         exitCode.set(currentExitCode)
-                        logger.info("Code: $currentExitCode")
+                        logger.info { "Code: $currentExitCode" }
                     }
                 }
             }
             threadPool.shutdown()
             threadPool.awaitTermination(1, TimeUnit.DAYS)
 
             val finalExitCode = exitCode.get()
-            logger.info("Code: $finalExitCode")
+            logger.info { "Code: $finalExitCode" }
             if (finalExitCode != 0) {
                 return finalExitCode
             }
             // Improvement: Try to extract merge commands before so user does not have to configure merge args?
-            logger.info("Each parser was successfully executed and created a cc.json file.")
-            return askAndMergeResults(commandLine)
+            return if (configuredParsers.size == 1) {
```

Comment:
I would find it easier to read if the return statement was inside of the if and else parts. The way it is currently it can potentially be confused with returning a function that consists if the if-else statement

Reply:
While I totally agree with you, I think this is the current standard way of doing it configured in Codecharta. If I moved the return statement into the if, I believe Sonar would throw a codesmell and ask me to move the return out of the if again which then might lead to us not reaching the quality gate.

What we should probably do is discuss to disable that rule since I think everyone on the team does not like that style :)